<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CO_Calculator</class>
 <widget class="QDialog" name="CO_Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cardiac Output Calculator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>570</y>
     <width>471</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="HUvalues">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>571</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
  </widget>
  <widget class="MatplotlibWidget" name="plotwidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="baselineInput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>500</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_baseline">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Baseline</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CO">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cardiac Output (L/min)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cardiacOutput">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_bloodHU">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blood Hounsfield Units</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tinterval">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Interval (s)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timeInterval">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="standardError">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>410</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_sterror">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>410</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>st. error</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="peakTime">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_peaktime">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time to Peak (s)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MTT">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_MTT">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>470</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean Transit Time (s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_timedensitycurve">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time Density Curve</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="CurveFitResults">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Curve Fit Results</string>
   </property>
   <widget class="QLabel" name="label_rsquared">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R^2</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="alpha">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="rsquared">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="beta">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AUC">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_beta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>beta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_alpha">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>alpha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_toffset">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>t0 (s)</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="t0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_AUC">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AUC</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_HUConversion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HU/Iodine (HU/(mg/mL)) </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="HUtoIodineConversion">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>540</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CO_Calculator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CO_Calculator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HUvalues</sender>
   <signal>entered(QModelIndex)</signal>
   <receiver>CO_Calculator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
